JS/Lesson14/savas/script.js

```diff
@@ -0,0 +1,104 @@
+console.log("Exercise 7.3: ", stringExercise7.charAt(stringExercise7.length - 3));
+
+console.log('------------------------');
+// 8. Use the `split` method to split a string into an array of substrings and print it to the console.
+const stringExercise8 = 'Algorithms are a fundamental part of computer science.';
+console.log("Exercise 8: ", stringExercise8.split(" "));
+
+console.log('------------------------');
+// 9. Use the `join` method to join an array of strings into a single string and print it to the console.
+const arrayExercise9 = ["An", "array",  "of", "strings"];
+console.log("Exercise 9: ", arrayExercise9.join("-"));
+
+console.log('------------------------');
+// 10. Use the `replace` method to replace "can help" with "help" in the string stringExercise10 and print the result to the console.
+const stringExercise10 = "Software design patterns can help simplify complex can help systems.";
+console.log("Exercise 10: ", stringExercise10.replace("can help", "help"));
+
+console.log('------------------------');
+// 11. Use the `startsWith` method to check if strings below starts with a price, and print the result to the console.
+const string1Exercise11 = "$9.99 - Sale! 50% off all items in store";
+const string2Exercise11 = "Bundle deal: Save $50 when you buy two products together";
+console.log("Exercise 11.1: ", string1Exercise11.startsWith("$"));
+console.log("Exercise 11.2: ", string2Exercise11.startsWith("$"));
+
+console.log('------------------------');
+// 12. Use the `endsWith` method to check if a string ends with a "!" (exclamation mark), and print the result to the console.
+const string1Exercise12 = "$9.99 - Sale! 50% off all items in store";
+const string2Exercise12 = "Bundle deal: Save $50 when you buy two products together!";
+console.log("Exercise 12.1: ", string1Exercise12.endsWith("!"));
+console.log("Exercise 12.2: ", string2Exercise12.endsWith("!"));
+
+
+console.log('------------------------');
+// 13. Use the `includes` method to check if a string includes a "%" character and print the result to the console.
+const string1Exercise13 = "$9.99 - Sale! 50% off all items in store";
+const string2Exercise13 = "Bundle deal: Save $50 when you buy two products together!";
+console.log("Exercise 13.1: ", string1Exercise13.includes("%"));
+console.log("Exercise 13.2: ", string2Exercise13.includes("%"));
+
+
+console.log('------------------------');
+// 14. Use the `substring` method to extract the word "Databases" from string stringExercise14 based on two indices and print it to the console.
+// Question: What is the difference between `substring()` and `slice()`?
+const stringExercise14 = 'Databases are an essential component of many applications.';
+console.log("Exercise 14a: ", stringExercise14.substring(0, 8));
+const stringExercise14b = 'The essential component of many applications are Databases';
+console.log("Exercise 14b: ", stringExercise14b.substring(stringExercise14b.indexOf("D"), stringExercise14b.length));
```

Comment:
Using `.indexOf("D")` is a bit dangerous because there could be other words starting with letter "D". Use the whole word instead:
```suggestion
console.log("Exercise 14b: ", stringExercise14b.substring(stringExercise14b.indexOf("Databases"), stringExercise14b.indexOf("Databases") + "Databases".length));

```

Reply:
Ok, I see the difference between them :)